file.go

```diff
@@ -23,3 +27,33 @@ func (f File) FilenameWithExtension() string {
 
 	return fmt.Sprintf("%s.%s", text, extension)
 }
+
+// AbsoluteFilePath returns a fake absolute path to a fake file (style is dependent on OS)
+func (f File) AbsoluteFilePath(levels int) string {
```

Comment:
You can use the other two AbsoluteFilePathFor* methods here and avoid to duplicate the logic.